src/mixins/itext_behavior.mixin.js

```diff
@@ -141,9 +141,15 @@
 
       this.abortCursorAnimation();
       this._currentCursorOpacity = 1;
-      this._cursorTimeout2 = setTimeout(function() {
-        _this._tick();
-      }, delay);
+      clearTimeout(this._cursorTimeout2);
+      if (delay) {
+        this._cursorTimeout2 = setTimeout(function () {
+          _this._tick();
+        }, delay);
+      }
+      else {
+        this._tick();
+      }
```

Comment:
i see how, with cursorDelay set to 0, the setTimeout isn't needed.
But with es6 syntax this would become 
```
his._cursorTimeout2 = setTimeout(() => this._tick(), delay);
```
And i would prefer it compare to an if.
Is the exta setTimout creating any kind of issue?

Reply:
As discussed this resolves a small flickering issue